### ISSUE TITLE
Fix background correction path problems

### ISSUE THREAD

**Issue by talonchandler** [mehta-lab/recOrder#349]
In my initial prototypes my background corrections strings weren't being passed correctly. After discussing with @ziw-liu and @edyoshikun I think I need to use `str` type and validate. I expect this to be a fairly shallow bug.

**Comment by edyoshikun** [comment]
something like this
```python 
from pydantic import BaseModel, validator, DirectoryPath

class _BirefringenceApplyInverseSettings(BaseModel):
background_path: str = ''
    @validator('background_path')
    def check_dir_path(cls, v):
        return DirectoryPath().validate(v)
```

**Comment by talonchandler** [comment]
Fixed by #353
